File: trabalho 1/VitorBasso11611BCC034/programas tarefa 1/java/micro/micro08.java
public class micro08{
    public static void main(String[] args){
        int numero;
        numero = 1;
        while (numero != 0){
            System.out.print("Digite um numero: ");
            numero = Integer.parseInt(System.console().readLine());
            if(numero > 10){
                System.out.println("O numero " + numero + " e maior que 10");
            }else{
                System.out.println("O numero " + numero + " e menor que 10");
            }
        }
    }
}

File: trabalho 6/codigos/lua/micro/micro08.lua
function main() int
    int numero
    numero = 1
    while numero ~= 0 do
        print("Digite um numero: ")
        numero = io.read()
        if numero > 10 then
            print("O numero ", numero, " e maior que 10\n")
        else
            print("O numero ", numero, " e menor que 10 \n")
        end
    end
    return 1
end

main()
